bulbs/liveblog/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+
+# from djes.models import Indexable
+
+from bulbs.content.models import Content
+from bulbs.liveblog.utils import (get_liveblog_author_model,
+                                  get_liveblog_model)
+
+
+class AbstractLiveBlog(models.Model):
+    """Base mixin inherited by each property's concrete LiveBlog implementation (along with base
+    content class).
+    """
+
+    recirc_content = models.ManyToManyField(Content, related_name='liveblog_recirc')
+
+    class Meta:
+        abstract = True
+
+
+class LiveBlogEntry(models.Model):
+
+    # Must import class (using string values for ForeignKeys doesn't work)
+    liveblog = models.ForeignKey(get_liveblog_model(), related_name='entries')
+
+    published = models.DateTimeField(blank=True, null=True)
+    authors = models.ManyToManyField(get_liveblog_author_model())
```

Comment:
or is this something we'll have to look out for in the event someone changes that model?